test/integration/api/datasets.js

```diff
@@ -1317,7 +1317,7 @@ describe('datasets and entities', () => {
               .then(({ headers, text }) => {
                 headers['content-disposition'].should.equal('attachment; filename="goodone.csv"; filename*=UTF-8\'\'goodone.csv');
                 headers['content-type'].should.equal('text/csv; charset=utf-8');
-                text.should.equal('name,label,first_name,age\n12345678-1234-4123-8234-123456789abc,Alice (88),Alice,88\n');
+                text.should.equal('name,label,version,first_name,age\n12345678-1234-4123-8234-123456789abc,Alice (88),1,Alice,88\n');
```

Comment:
One thing I noticed when trying to make a form that looked up the version, and based on what we're talking about in the team meeting on Aug 29, `name` and `label` have to be what they are, but should `version` be `__version`?

Reply:
I like `version` for the attachment csv